File: Examples/AspNetMvcVB/Models/VideoInfoModel.vb
Namespace Models
    Public Class VideoInfoModel
        Public Sub New()
            Properties = New Dictionary(Of String, String)()
            Metadata = New Dictionary(Of String, String)()
        End Sub

        Public ReadOnly Property Properties() As Dictionary(Of String, String)

        Public ReadOnly Property Metadata() As Dictionary(Of String, String)
    End Class
End NameSpace

File: Examples/AspNetMvcCS/Models/VideoInfoModel.cs
using System.Collections.Generic;

namespace GleamTech.VideoUltimateExamples.AspNetMvcCS.Models
{
    public class VideoInfoModel
    {
        public VideoInfoModel()
        {
            Properties = new Dictionary<string, string>();
            Metadata = new Dictionary<string, string>();
        }

        public Dictionary<string, string> Properties { get; }

        public Dictionary<string, string> Metadata { get; }
    }
}